### PR TITLE
Add edit deploy location and fix defaults

### PR DESCRIPTION
**Pull request by SanjulaGanepola**
### Changes
- Add `Edit Deploy Location` on Source tree item
- Fix defaults for when prompting for deploy location

### CODE REVIEW

File: src/views/projectExplorer/project.ts [hunk 1]
```diff
@@ -41,12 +41,15 @@ export default class Project extends ProjectExplorerTreeItem {
     if (remoteDir) {
       items.push(new Source(this.workspaceFolder, remoteDir));
     } else {
+      const homeDirectory = (ibmi?.getConfig().homeDirectory.endsWith('/') ? ibmi?.getConfig().homeDirectory.slice(0, -1) : ibmi?.getConfig().homeDirectory);
+      const defaultDeployLocation = `${homeDirectory}/${this.workspaceFolder.name}`;
```
[the remark this answers is not in the document]
Comment (SanjulaGanepola) [reply src/views/projectExplorer/project.ts]: Fixed

**Comment by edmundreinhardt** [comment]
Perhaps this has been addressed, when I added a new folder to the workspace, the project explorer went blank and said that I should connect ot an IBM i, but I was already connected to an IBM i with the previous project - should I create a project?

**Review by edmundreinhardt — COMMENTED** [review]
if you just answer my one question

**Review by edmundreinhardt — COMMENTED** [review]
if you just answer my one question

**Comment by edmundreinhardt** [comment]
Note that in this case there is no way to reconnect to an IBM i, the Code for IBM i is not connected but doesn't show the connection list.  No way to connect/reconnect on status bar as well
@worksofliam

**Comment by SanjulaGanepola** [comment]
In VS Code, if you change the first workspace folder or go from a single folder workspace to a multi-root workspace, all the extensions are restarted. For Code for i, there is a known issue that when this happens the connection view appears as you saw.

**Review by edmundreinhardt — APPROVED** [review]

**Review by edmundreinhardt — APPROVED** [review]

**Pull request merged**